utils/hooks/useFetchEarth.js

```diff
@@ -0,0 +1,43 @@
+import { createApolloFetch } from "apollo-fetch";
+import { useState, useEffect } from "react";
+
+const fetch = createApolloFetch({
+  uri: 'https://countries.trevorblades.com/graphql',
+});
+
+/**
+ * useFetchEarth.
+ *
+ * @param {string} query in graphql to be fetch the data
+ */
+export function useFetchEarth(query) {
+
+  const [data, setData] = useState({});
+  const [loading, setLoading] = useState(false);
+  const [error, setError] = useState(null);
+
+  async function fetchData() {
```

Comment:
Not sure how to get this function in code coverage

Reply:
Man I'm sorry I missed this comment as well. did you find a way to test it? if not let's go over it together. Can you put a short block on my calendar?